### PR TITLE
feat: added input component

### PR DESCRIPTION
**Pull request by sks-21**
Added **Input** component 
Includes

- Standard Input component
- Input with icons
- Input with error validation

### CODE REVIEW

File: components/input/input.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Futbolt UI | Input</title>
+    <script src="https://kit.fontawesome.com/0a41da8839.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/components/input/input.css" />
+</head>
+
+<body>
+    <!-- || Container  -->
+    <div class="container">
+
```
Comment (Kothalkarpriya) [inline components/input/input.html]: remove extra space

File: components/input/input.html [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                </ul>
+            </div>
+        </nav>
+
+        <!-- || Side Menu starts   -->
+        <aside class="menu flx-column flx-main-center">
+            <ul class="menu-list">
+                <li class="menu-list-title">Getting Started</li>
+                <li>
+                    <a href="">Installation</a>
```
Comment (Kothalkarpriya) [inline components/input/input.html]: use # to make dead link

File: components/input/input.html [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+                </li>
+                <li>
+                    <a href="">Typography</a>
+                </li>
+                </li>
+                <li>
+                    <a href="">Colors</a>
```
Comment (Kothalkarpriya) [inline components/input/input.html]: here also

File: components/input/input.html [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+                    </div>
+
+                </form>
+
+                <br>
+                <br>
+
+                <!-- || Content footer starts  -->
+                <div class="content-footer flx-row">
+                    <a href="" class="content-footer-prev"><i class="fas fa-angle-left"></i> Image
```
Comment (Kothalkarpriya) [inline components/input/input.html]: use #

File: styles.css [hunk 5]
```diff
@@ -126,34 +133,44 @@
...
   background-color: var(--color-red-dark);
 }
 
 .bg-yellow-hover {
   background-color: var(--color-yellow);
 }
+
 .bg-yellow-hover:hover {
   background-color: #eab308;
 }
```
Comment (Kothalkarpriya) [inline styles.css:147]: define value in root and use it with as before you have used

**Review by Kothalkarpriya — CHANGES_REQUESTED** [review]
Look into my comments

**Review by Kothalkarpriya — APPROVED** [review]

**Review by mariyasada — APPROVED** [review]
looks good

File: components/input/input.css [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    height: 5rem;
+}
+
+.form-input-text {
+    border: 2px solid var(--color-gray-light);
+}
+
+.form-input:focus {
+    border: 2px solid var(--color-secondary);
+    outline: none;
```
Comment (JantuDeb) [inline components/input/input.css:30]: Don't do `outline: none`  if you want to hide outline then make it transparent `outline: 2px solid transparent;`

File: components/input/input.css [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+.form-textarea {
+    font-family: inherit;
+    border: 2px solid var(--color-gray-light);
+    border-radius: 5px;
+    padding: 8px;
+}
+
+.form-textarea:focus {
+    outline: none;
```
Comment (JantuDeb) [inline components/input/input.css:41]: Same as above

File: styles.css [hunk 8]
```diff
@@ -181,31 +199,45 @@
 .flx-cross-center {
   justify-content: center;
 }
+
 /* || Spacing */
-.vertical-gutter-lg{
-margin-top: 2.4rem;
+.vertical-gutter-lg {
+  margin-top: 2.4rem;
```
Comment (JantuDeb) [inline styles.css]: `margin-top-lg` would be a great choice for this class name.

File: styles.css [hunk 9]
```diff
@@ -181,31 +199,45 @@
...
+  border-radius: 5px;
+}
+
+.brad-10 {
+  border-radius: 10px;
+}
+
+.brad-15 {
+  border-radius: 15px;
+}
```
Comment (JantuDeb) [inline styles.css:227]: Use `border-radius-*` or `radius-*`  `brad-*` is confusing name for `border-radius`

**Review by JantuDeb — APPROVED** [review]
Suggested some changes.

**Pull request merged**